src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="post"
```

Comment:
The `action` attribute should contain a URL or path to a server endpoint. If you intended to specify the HTTP method, use the `method` attribute with the value `post` instead.